From: Vincent Bernat <vincent@bernat.im>
Date: Thu, 7 Dec 2017 15:55:13 +0100
Subject: Remove upstream Debian-related stuff

---
 CMakeLists.txt | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 05d4d03..b26b449 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,15 +30,3 @@ endif()
 
 # Add the Server directory
 add_subdirectory (Server)
-
-FIND_PROGRAM(DPKG_CMD dpkg)
-IF(DPKG_CMD)
-  add_subdirectory (deb_package)
-ENDIF(DPKG_CMD)
-
-FIND_PROGRAM(RPKG_CMD rpmbuild)
-IF(RPKG_CMD)
-  add_subdirectory (rpm_package)
-ENDIF(RPKG_CMD)
-
-
